### ISSUE TITLE
Operation not permitted preserving times results in myapp container stopping under Windows

### ISSUE THREAD

**Issue by LSaraguA** [bitnami/bitnami-docker-laravel#102]
**Description**

MyApp container stops while starting environment on Windows.

**Steps to reproduce the issue:**

1. Download _docker-compose.yml_ file
2. Run `docker-compose up`

**Describe the results you received:**

Messages shown in log:
```
myapp_1    | cp: preserving times for '/app/./vendor/bin/php-parse': Operation not permitted
myapp_1    | cp: preserving times for '/app/./vendor/bin/phpunit': Operation not permitted
myapp_1    | cp: preserving times for '/app/./vendor/bin/psysh': Operation not permitted
myapp_1    | cp: preserving times for '/app/./vendor/bin/var-dump-server': Operation not permitted
...
laravel_myapp_1 exited with code 1
```

**Describe the results you expected:**

MyApp container runs.

**Additional information you deem important (e.g. issue happens only occasionally):**

N/A

**Version**

- Output of `docker version`:

```
Client: Docker Engine - Community
 Version:           18.09.2
 API version:       1.39
 Go version:        go1.10.8
 Git commit:        6247962
 Built:             Sun Feb 10 04:12:31 2019
 OS/Arch:           windows/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          18.09.2
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.6
  Git commit:       6247962
  Built:            Sun Feb 10 04:13:06 2019
  OS/Arch:          linux/amd64
  Experimental:     false
```

- Output of `docker info`:

```
Containers: 7
 Running: 7
 Paused: 0
 Stopped: 0
Images: 138
Server Version: 18.09.2
Storage Driver: overlay2
 Backing Filesystem: extfs
 Supports d_type: true
 Native Overlay Diff: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
 Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 9754871865f7fe2f4e74d43e2fc7ccd237edcbce
runc version: 09c8266bf2fcf9519a651b04ae54c967b9ab86ec
init version: fec3683
Security Options:
 seccomp
  Profile: default
Kernel Version: 4.9.125-linuxkit
Operating System: Docker for Windows
OSType: linux
Architecture: x86_64
CPUs: 2
Total Memory: 1.934GiB
Name: linuxkit-00155d102d0d
ID: 4FGP:BGCJ:KORU:IMEQ:UGRT:RKVG:WYCE:5GHZ:24KG:T7Q7:KHZC:T3P6
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): true
 File Descriptors: 98
 Goroutines: 126
 System Time: 2019-05-27T12:06:32.6222005Z
 EventsListeners: 4
Registry: https://index.docker.io/v1/
Labels:
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
Product License: Community Engine
```

- Output of `docker-compose version` (if applicable):

```
docker-compose version 1.23.2, build 1110ad01
docker-py version: 3.6.0
CPython version: 3.6.6
OpenSSL version: OpenSSL 1.0.2o  27 Mar 2018
```

**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**

N/A

**Comment by LSaraguA** [comment]
Proposed change (current fix in my environment):

```
    privileged: true
    
```

**Comment by andresbono** [comment]
Hi @LSaraguA,

Thanks for reporting the issue and sharing the solution that worked for you. We just replied in the PR.

**Comment by LSaraguA** [comment]
Fixed in #103
